cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -128,7 +128,7 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Eager Aggregation",
       text =
-        """Eagerly loads underlying results and stores it in a hash-map, using the grouping keys as the keys for the map.""".stripMargin,
+        """The `EagerAggregation` operator eagerly loads underlying results and stores it in a hash map, using the grouping keys as the keys for the map.""".stripMargin,
```

Comment:
This is a very mechanical explanation for what is happening. What about:

The `EagerAggregation` operator uses the grouping columns to group rows into different groupings. For each of these groupings, this operator will then evaluate all aggregation functions and return the result. 
To do this, the operator, as the name implies, needs to eagerly pull in all data from its source and hold it in the query state, which leads to added memory pressure in the system.

Reply:
👍